src/main/services/catalog.ts

```diff
@@ -241,7 +242,23 @@ export class CatalogService {
         return await this.publicationRepository.save(newPub);
     }
 
-    public exportPublication(publication: PublicationView, destinationPath: string) {
+    public async exportPublication(publication: PublicationView) {
+        // Get main window
+        const winRegistry = diMainGet("win-registry");
+        let mainWindow;
+        for (const window of (Object.values(winRegistry.getWindows())) as any) {
+            if (window.type === "library") {
+                mainWindow = window;
+            }
+        }
+
+        // Open a dialog to select a folder then copy the publication in it
+        let destinationPath: string = dialog.showOpenDialog(mainWindow, {
+            properties: ["openDirectory"],
+        })[0];
+
+        // If the selected path is a file then choose the directory containing this file
+        destinationPath = path.dirname(destinationPath);
```

Comment:
This code fails to achieve the desired result. Tests:

`node -e "const path = require('path'); console.log(path.dirname('/folder1/folder2/file1.txt'));"`
=>
`/folder1/folder2`
(correct)


`node -e "const path = require('path'); console.log(path.dirname('/folder1/folder2'));"`
=>
`/folder1`
(incorrect, expected result is `/folder1/folder2`)